LPS-149052 frontend-data-set-web: Remove quotes around 'Items Selected'

oliv-yu opened the pull request:
https://issues.liferay.com/browse/LPS-149052 - Remove quotes around 'Items Selected' when using bulk action on FDS display

Small change, thanks for taking a look!

Before:
<img width="1421" alt="Screen Shot 2022-03-10 at 2 54 01 PM" src="https://user-images.githubusercontent.com/14063502/157771023-0b7740bc-2cf1-45a1-9872-9199a7a61d6c.png">
After:
<img width="1396" alt="Screen Shot 2022-03-10 at 3 16 35 PM" src="https://user-images.githubusercontent.com/14063502/157771291-a4ede993-b94b-4573-9a39-07ca6855cd18.png">

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/2464/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2464/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-source-format/2052/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/2052/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4dbd5e4e15ea41326bd5a809d4570e6f0b395ea2">4dbd5e4e15ea41326bd5a809d4570e6f0b395ea2</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPS-149052-items-selected-quotes">LPS-149052-items-selected-quotes</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/4872bf44d3ba3626a304e34baaacc1b99c325ef6">4872bf44d3ba3626a304e34baaacc1b99c325ef6</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-source-format/2052/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/2052/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-source-format/2052/">test-portal-source-format#2052</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/2052/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1996">liferay-frontend#1996</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1632083929">[master] ci:test:sf - oliv-yu &gt; liferay-frontend - PR#1996 - 2022-03-10[15:17:46]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/7312/">publish-testray-report#7312</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 26 out of 27 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 27 out of 28 jobs passed in 1 hour 54 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4dbd5e4e15ea41326bd5a809d4570e6f0b395ea2">4dbd5e4e15ea41326bd5a809d4570e6f0b395ea2</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8ff76412b3a7b8bd27f0ade1c876eff429eed7f3">8ff76412b3a7b8bd27f0ade1c876eff429eed7f3</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//3015">Acceptance Upstream DXP (master) #3015</a></p><details><summary><strong>ci:test:stable - 26 out of 27 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39061/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>26 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40070/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339713/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40074/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39059/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40068/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39060/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40073/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25023/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/511271/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339717/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33568/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431978/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33569/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33567/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339716/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339715/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40071/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40069/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401704/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/473706/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40072/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33566/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401702/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339714/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431977/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401703/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 26 out of 28 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/2464/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39061/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>26 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40070/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339713/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40074/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39059/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40068/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39060/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40073/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/25023/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/511271/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339717/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33568/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431978/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33569/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33567/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339716/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339715/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40071/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40069/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401704/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/473706/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40072/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33566/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401702/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339714/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431977/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401703/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2464/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 8ff76412b3a7b8bd27f0ade1c876eff429eed7f3:</strong></summary><ol><li><div><h5><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39061/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/39061//consoleText">AXIS_VARIABLE=0 #39061</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code>  [waitfor]   at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:9.0.56]
  [waitfor]   at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) [catalina.jar:9.0.56]
  [waitfor]   at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:9.0.56]
  [waitfor]   at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357) [catalina.jar:9.0.56]
  [waitfor]   at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382) [tomcat-coyote.jar:9.0.56]
  [waitfor]   at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:9.0.56]
  [waitfor]   at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895) [tomcat-coyote.jar:9.0.56]
  [waitfor]   at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732) [tomcat-coyote.jar:9.0.56]
  [waitfor]   at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:9.0.56]
  [waitfor]   at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:9.0.56]
  [waitfor]   at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:9.0.56]
  [waitfor]   at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:9.0.56]
  [waitfor]   at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
  [waitfor] Caused by: java.lang.NullPointerException
  [waitfor]   at com.liferay.portal.events.ServicePreAction._initThemeDisplay(ServicePreAction.java:1342) ~[portal-impl.jar:?]
  [waitfor]   at com.liferay.portal.events.ServicePreAction.servicePre(ServicePreAction.java:173) ~[portal-impl.jar:?]
  [waitfor]   at com.liferay.portal.events.ServicePreAction.run(ServicePreAction.java:154) ~[portal-impl.jar:?]
  [waitfor]   ... 150 more
  [waitfor] 2022-03-11 00:35:22.743 ERROR [http-nio-8080-exec-6][error_jsp:563] User ID null
  [waitfor] 2022-03-11 00:35:22.744 ERROR [http-nio-8080-exec-6][error_jsp:564] Current URL /web/guest
  [waitfor] 2022-03-11 00:35:22.744 ERROR [http-nio-8080-exec-6][error_jsp:565] Referer null
  [waitfor] 2022-03-11 00:35:22.744 ERROR [http-nio-8080-exec-6][error_jsp:566] Remote address 127.0.0.1
  [waitfor] 2022-03-11 00:35:22.744 ERROR [http-nio-8080-exec-6][error_jsp:574] Exception is null</code></pre></div></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/2464/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/2464/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/2464/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/2464/">test-portal-acceptance-pullrequest(master)#2464</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2464/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1996">liferay-frontend#1996</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1632178204">[master] ci:test:relevant - oliv-yu &gt; liferay-frontend - PR#1996 - 2022-03-10[15:17:41]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/7317/">publish-testray-report#7317</a><br/></div></div>

diegonvs commented on modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/BulkActions.js:
```diff
@@ -165,7 +165,9 @@ function BulkActions({
 									} ${Liferay.Language.get(
 										'of'
-									)} ${total} Liferay.Language.get('items-selected')`}
+									)} ${total} ${Liferay.Language.get(
```
Hey @oliv-yu, when taking a look on https://liferay.design/lexicon/core-components/toolbars/management-bar/, looks like it should be lowercase.

Also, why this language key isn't: `{0}-of-{1}-items-selected`? Then, you can use `sub` function there like `Liferay.Util.sub(Liferay.Language.get({0}-of-{1}-items-selected), selectedItemsValue.length, total)`

oliv-yu commented:
Thanks, @diegonvs! You're right, I added that change just now. The only thing was that there is no language key for `x-of-x-items-selected` so I used `x-of-x-selected` instead.

Here's what it looks like now: 
<img width="1433" alt="Screen Shot 2022-03-11 at 9 00 18 AM" src="https://user-images.githubusercontent.com/14063502/157915482-8f7a9ac8-1cc2-44d3-9464-382d9463ed10.png">

diegonvs commented:
Hey @oliv-yu, for preventing a new issue opened to change the text again -- I have a vague memory that I've already worked things like this -- I would recommend you to create a new language key.

diegonvs commented:
Sorry for the nitpicking 🥺

oliv-yu commented:
No problem, @diegonvs! I appreciate the wisdom haha. Added the language key (uncapitalized) into `Language.properties` and rebased off master. 

New screenshot:
<img width="1400" alt="Screen Shot 2022-03-11 at 9 54 03 AM" src="https://user-images.githubusercontent.com/14063502/157923751-b390bf0d-24ff-4c19-9ad0-f7da05c1b7ef.png">

oliv-yu commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-source-format/2404/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/2404/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4bc7f074155fae13178dc677ef75b5c25b60fe4d">4bc7f074155fae13178dc677ef75b5c25b60fe4d</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPS-149052-items-selected-quotes">LPS-149052-items-selected-quotes</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/292ba3c6d1abed2ff7fea4b285c9dc43b7536446">292ba3c6d1abed2ff7fea4b285c9dc43b7536446</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-source-format/2404/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/2404/jenkins-report.html">here</a>.</h5></details></html>

oliv-yu commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/2189/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2189/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-source-format/2404/">test-portal-source-format#2404</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/2404/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1996">liferay-frontend#1996</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1632999340">[master] ci:test:sf - oliv-yu &gt; liferay-frontend - PR#1996 - 2022-03-11[11:59:19]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/8755/">publish-testray-report#8755</a><br/></div></div>

diegonvs reviewed (approved):
![capy-approves](https://c.tenor.com/fvm92ITb-YEAAAAd/farthyun-capybara.gif)

Capy approves

oliv-yu commented:
ci:forward

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-28.liferay.com/job/forward-pullrequest/295//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/114578
[Console](https://test-1-28.liferay.com/job/forward-pullrequest/295//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/2189/">test-portal-acceptance-pullrequest(master)#2189</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2189/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1996">liferay-frontend#1996</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1633153488">[master] ci:test:relevant - oliv-yu &gt; liferay-frontend - PR#1996 - 2022-03-11[12:04:02]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/6258/">publish-testray-report#6258</a><br/></div></div>
